Build/Packages/permission-sets-examples/Configuration/PermissionSets/some-web-modules.yaml

```diff
@@ -0,0 +1,3 @@
+label: Allow web_info modules
```

Comment:
label should be "Allow web_page and web_list modules"

Reply:
True, fixed it.